pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-parent</artifactId>
-    <version>3.1.2</version>
+    <version>3.2.4</version>
```

Comment:
Ugh... This lib depends on Spring?! 🙁 It looks like the only spring things actually getting used are some annotations on some service classes (and dependency injection for those services), which arguably all belong in mod-fqm-manager. I can't wait for us to finally get rid of this stupid lib... 🙁